Sprint-1/interpret/to-pounds.js

```diff
@@ -1,27 +1,36 @@
 const penceString = "399p";
 
 const penceStringWithoutTrailingP = penceString.substring(
-  0,
-  penceString.length - 1
-);
+  0, // extracts a portion of the string here starting from index 0
+  penceString.length - 1 // extract ccarcter p (-1) index
+); // the result is asing to penceStringWithoutTrailingP
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); // check it the string has 3 characters
 const pounds = paddedPenceNumberString.substring(
-  0,
-  paddedPenceNumberString.length - 2
-);
+  0, //and stop with (0) so has to be 3 characters
+  paddedPenceNumberString.length - 2 //gets the characters from the beginning of the string up to 2 characters from the end.
+); // result is stored in paddedPenceNumberString
 
 const pence = paddedPenceNumberString
-  .substring(paddedPenceNumberString.length - 2)
-  .padEnd(2, "0");
+  .substring(paddedPenceNumberString.length - 2) // extracts the last 2 characters from the strin
+  .padEnd(2, "0"); //ensures the string is at least 2 characters long
```

Comment:
Can we expect this program to work as intended even if we deleted `.padEnd(2, "0")` from the code?